recipe/meta.yaml

```diff
@@ -142,13 +138,31 @@ requirements:
     # - azure-identity >=1.0.0
     # - airflow >=1.9.0
     # - Click >=7.1.2
+    # - sqlalchemy >=1.3.16
+    # - snowflake-sqlalchemy >=1.2
+    # - ruamel_yaml <0.17.18,>=0.16
```

Comment:
Ah, this matching failed!

Reply:
oh is it automated or something?

Reply:
Yes, the cf-scripts extract the requirements from the whl (if there is one) if you have set `bot.run_deps_from_wheel: true` in the conda_forge.yml.